fusionfrontend/src/app/store/factory-asset-details/factory-asset-details.model.ts

```diff
@@ -29,6 +29,8 @@ export class FactoryAssetDetails extends Asset {
   openAlertPriority: OispAlertPriority;
   statusHoursOneDay?: StatusHoursOneDay;
   protocol: string;
+  daysTillMaintenance: number;
+  operatingHoursTillMaintenance: number;
```

Comment:
should we really include fields of an asset in the model? Not sure whether this is a good idea.